spec/controllers/api/reporting/totals_controller_spec.rb

```diff
@@ -121,4 +121,271 @@
       expect(csv.length).to eq(2)
     end
   end
+
+  describe "Dashboard acceptance criteria" do
+    let(:team) { Team.last }
+    let(:programme) { create(:programme, :hpv, teams: [team]) }
+    let(:session) { create(:session, team:, programmes: [programme]) }
+    let(:flu_programme) { create(:programme, :flu, teams: [team]) }
+    let(:flu_session) { create(:session, team:, programmes: [flu_programme]) }
+
+    let(:cohort) { parsed_response["cohort"] }
+    let(:vaccinated) { parsed_response["vaccinated"] }
+    let(:not_vaccinated) { parsed_response["not_vaccinated"] }
+    let(:vaccinated_by_sais) { parsed_response["vaccinated_by_sais"] }
+    let(:vaccinated_elsewhere_declared) do
+      parsed_response["vaccinated_elsewhere_declared"]
+    end
+    let(:vaccinated_elsewhere_recorded) do
+      parsed_response["vaccinated_elsewhere_recorded"]
+    end
+    let(:vaccinations_given) { parsed_response["vaccinations_given"] }
+    let(:monthly_vaccinations_given) do
+      parsed_response["monthly_vaccinations_given"]
+    end
+
+    def refresh_and_get_totals(programme_type: "hpv")
+      ReportingAPI::PatientProgrammeStatus.refresh!(concurrently: false)
+      get :index, params: { programme: programme_type }
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "AC1: child vaccinated by SAIS" do
+      patient = create(:patient, session:)
+      create(
+        :vaccination_record,
+        patient:,
+        programme:,
+        session:,
+        outcome: "administered",
+        performed_at: Time.current
+      )
+
+      refresh_and_get_totals
+
+      expect(cohort).to eq(1)
+      expect(vaccinated).to eq(1)
+      expect(not_vaccinated).to eq(0)
+      expect(vaccinated_by_sais).to eq(1)
+      expect(vaccinations_given).to eq(1)
+
+      monthly =
+        monthly_vaccinations_given.find do |m|
+          m["year"] == Time.current.year &&
+            m["month"] == Date::MONTHNAMES[Time.current.month]
```

Comment:
```suggestion
        monthly_vaccinations_given.find do
          it["year"] == Time.current.year &&
            it["month"] == Date::MONTHNAMES[Time.current.month]
```

I don't think `m` is any more descriptive than `it`. 😄